src/js/stores/__tests__/CosmosPackagesStore-test.js

```diff
@@ -69,7 +69,7 @@ describe("CosmosPackagesStore", function() {
       it("stores availablePackages when event is dispatched", function() {
```

Comment:
I also think we should provide an integration test.

Reply:
👍